app/admin.go

```diff
@@ -59,6 +60,10 @@ func (s *Server) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError)
 	var lines []string
 
 	if *s.Config().LogSettings.EnableFile {
+		timeoutCtx, timeoutCancel := context.WithTimeout(context.Background(), time.Second*5)
```

Comment:
a constant instead of a hardcoded timeout would be better I think.

Reply:
Thanks.  Fixed https://github.com/mattermost/mattermost-server/pull/15398/commits/ebdd80bd4f8b76eaaeb3862fe021bbe1c30fecdc